frontend/src/forms/ParcelDetailForm.test.tsx

```diff
@@ -86,103 +87,199 @@ const parcelDetailForm = (
     </Router>
   </Provider>
 );
+describe('ParcelDetailForm', () => {
+  describe('field validation', () => {
+    const exampleData = {
+      pid: '',
+      agencyId: 1,
+      address: {
+        line1: 'addressval',
+        cityId: 1,
+        provinceId: '2222',
+        postal: 'V8X3L5',
+      },
+      description: '',
+      landLegalDescription: '',
+      pin: '5',
+      zoning: 'zoningVal',
+      zoningPotential: 'zoningPotentialVal',
+      municipality: 'municipalityVal',
+      landArea: 1234,
+      statusId: 1,
+      classificationId: 1,
+      isSensitive: false,
+      latitude: 0,
+      longitude: 0,
+      evaluations: [],
+      buildings: [],
+    };
+    const fillInput = async (
+      container: HTMLElement,
+      name: string,
+      value: any,
+      type: string = 'input',
+    ) => {
+      const input = container.querySelector(`${type}[name="${name}"]`);
+      await wait(() => {
+        if (type === 'input') {
+          fireEvent.change(input!, {
+            target: {
+              value: value,
+            },
+          });
+        } else {
```

Comment:
I wonder if the `else` branch is needed here since they both execute the same `fireEvent,change()` code. Am I missing something subtle here?

Reply:
No, I was messing around with this and had an alternate branch at some point, I'll move this into a helper class and fix the if/else in my next PR.